packages/perseus/src/widgets/input-number/input-number-validator.test.ts

```diff
@@ -0,0 +1,262 @@
+                images: {},
+                widgets: {
+                    "input-number 1": {
+                        type: "input-number",
+                        graded: true,
+                        options: {
+                            maxError: 0.1,
+                            inexact: false,
+                            value: 0.5,
+                            simplify: "optional",
+                            answerType: "percent",
+                            size: "small",
+                        },
+                    },
+                },
+            } as PerseusRenderer,
+            "50%",
+            "0.56",
+        ],
+    ])("answer type", (question, correct, incorrect) => {
+        it("Should accept the right answer", async () => {
+            // Arrange
+            const {renderer} = renderQuestion(question);
+
+            // Act
+            const textbox = screen.getByRole("textbox");
+            await userEvent.click(textbox);
+            await userEvent.type(textbox, correct);
+
+            // Assert
+            expect(renderer).toHaveBeenAnsweredCorrectly();
+        });
+
+        it("should reject an incorrect answer", async () => {
+            // Arrange
+            const {renderer} = renderQuestion(question);
+
+            // Act
+            const textbox = screen.getByRole("textbox");
+            await userEvent.click(textbox);
+            await userEvent.type(textbox, incorrect);
+
+            // Assert
+            expect(renderer).toHaveBeenAnsweredIncorrectly();
+        });
+    });
+
+    it("transform should remove the `value` field", function () {
+        const editorProps = {
+            value: 5,
+            simplify: "required",
+            size: "normal",
+            inexact: false,
+            maxError: 0.1,
+            answerType: "number",
+        } as const;
+        if (!transform) {
+            throw new Error("transform not defined");
+        }
+        const widgetProps = transform(editorProps, mockStrings);
+        expect(_.has(widgetProps, "value")).toBe(false);
+    });
```

Comment:
This isn't related to scoring, but I didn't want to have all of the item info in two separate places. Feel free to let me know if that is preferred to having this test here though. Thanks!